### ISSUE TITLE
Transliterating Serbian names (Cyrillic to Latin)

### ISSUE THREAD

**Issue by djbpitt** [djbpitt/repertorium#11]
Andrej writes:

> Should'n we have Matić instead of Matich?

David replied:

>If we're rendering a Serbian name in Latin letters I think we should use the BCS spelling, so yes, Matić. The decision becomes difficult, though, because people publish under different Romanizations of the same name. What if there's some American named Matich who publishes in English and spells his name "Matich"?

Andrej replied:

>I would propose if it is a Serbian/Russian/Bulgarian name and we should transliterate it, then to follow http://repertorium.obdurodon.org/transliteration.xhtml (as you suggest), otherwise we should spell the name as the author prefers. So in this particular case will be Matić, because it is a transliteration.

New response (David):

I agree with Andrej's proposal above. At some point we may run into situations where authors have published with their names in Latin letters and in Cyrillic, and in inconsistent ways; cf. Čiževskij or Isačenko. We may want to revisit this question should we run into authors in our materials whose names pose that sort of problem. For now, though, I agree that if the name on the original work is spelled "Матић", we should write "Matić".

I will move this card to the **To be implemented** column. It does not require a change in ODD or schema, but it requires revising the XML documents and then my revising the addendum to our markup guidelines.

**Comment by atoboy** [comment]
Agree. We should not forget to include this decision in our Guidelines.
